api/src/main/java/com/codeforcommunity/dto/emailer/EmailAttachment.java

```diff
@@ -0,0 +1,111 @@
+package com.codeforcommunity.dto.emailer;
+
+import com.codeforcommunity.aws.EncodedImage;
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.BadRequestImageException;
+import com.codeforcommunity.exceptions.HandledException;
+
+import org.simplejavamail.api.email.AttachmentResource;
+
+import javax.mail.util.ByteArrayDataSource;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+
+public class EmailAttachment extends ApiDto {
+    private String name;
+    private String data;
+
+    public EmailAttachment(String name, String data) {
+        this.name = name;
+        this.data = data;
+    }
+
+    private EmailAttachment() {}
+
+    private static String getFileExtension(String base64Image) {
+        // Expected Base64 format: "data:image/{extension};base64,{imageData}"
+
+        if (base64Image == null || base64Image.length() < 10) {
```

Comment:
If the encoding length is 11 will it still be valid? If this is based on legacy code that's fine, just wondering if there's other ways to check image validity

Reply:
yeah this is based on the s3requester legacy code, unsure of the reason behind 10